packages/client/mutations/CreateTaskMutation.ts

```diff
@@ -50,6 +52,9 @@ graphql`
         ... on _xGitHubIssue {
           number
           title
+          repository {
```

Comment:
I'd love to use the `@appendNode` directive, but that requires a connectionId that is specific to the viewer. since our subscriptions share the same fragment, i'm not sure how best to do that declaratively